### PR TITLE
Add base onewallet module and linking sample app

### PR DESCRIPTION
**Pull request by berryve**

### CODE REVIEW

File: onewallet/src/main/java/org/kinecosystem/linkwallet/LinkAccountButton.kt [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import android.widget.Button
+import android.widget.TextView
+
+
+class LinkAccountButton @JvmOverloads constructor(context: Context, attrs: AttributeSet? = null, defStyleAttr: Int = 0) :
+        Button(context, attrs, defStyleAttr) {
+
+    init {
+        text = "Link Account" //resources.getString(R.string.explore_kin)
+        //TextViewCompat.setTextAppearance(this, R.style.kinTextButtonRounded_Purple)
```
Comment (katzio) [inline onewallet/src/main/java/org/kinecosystem/linkwallet/LinkAccountButton.kt]: remove comments

File: transfer/src/main/java/org/kinecosystem/transfer/sender/manager/TransferManager.kt [hunk 2]
```diff
@@ -18,49 +17,41 @@ class TransferManager(var activity: Activity?) {
...
+        }
+
+        fun start(requestCode: Int): Boolean {
+            activity?.let {
+                val packageManager = it.packageManager
+                intent.component = ComponentName(applicationId, launchActivityFullPath)
+                val resolveInfos = packageManager.queryIntentActivities(intent, 0)
+                if (!resolveInfos.isEmpty()) {
+                    if (resolveInfos[0].activityInfo.exported) {
+                        val appName = activity.applicationInfo.loadLabel(activity.packageManager).toString()
```
Comment (katzio) [inline transfer/src/main/java/org/kinecosystem/transfer/sender/manager/TransferManager.kt]: try catch?

**Review by katzio — APPROVED** [review]

File: appsDiscovery/src/main/java/org/kinecosystem/appsdiscovery/presenter/AppInfoPresenter.kt [hunk 3]
```diff
@@ -153,8 +154,8 @@ class AppInfoPresenter(private val appName: String?, private val repository: Dis
         repository.clearReceiverAppPublicAddress()
         app?.launchActivity?.let { activityPath ->
             app?.identifier?.let { receiverPkg ->
-                val started = transferManager.startTransferRequestActivity(REMOTE_PUBLIC_ADDRESS_REQUEST_CODE,
-                        receiverPkg, activityPath)
+                val started = transferManager.requestBuilder(receiverPkg, activityPath)
```
Comment (shayb) [inline appsDiscovery/src/main/java/org/kinecosystem/appsdiscovery/presenter/AppInfoPresenter.kt]: lets call start and build

Comment (shayb) [inline appsDiscovery/src/main/java/org/kinecosystem/appsdiscovery/presenter/AppInfoPresenter.kt]: intent request 
transfer builder?

**Review by shayb — APPROVED** [review]
please look at my comments
and please check that transfer is not broken some how

**Pull request merged**